airgun/entities/computeresource.py

```diff
@@ -37,6 +37,7 @@ def delete(self, value):
     def list_vms(self, rhev_name, expected_vm_name=None):
         """Returns all the VMs on the CR or VM with specified name"""
         view = self.navigate_to(self, 'Detail', rhev_name=rhev_name)
+        view.virtual_machines.fill({'vm_search': expected_vm_name})
```

Comment:
I have feeling even in old version two different functionalities mixed in one method: search and list_all. After that change it become even more obvious that we should distinguish these methods